src/components/editor/PropertyTemplate.jsx

```diff
@@ -35,5 +36,14 @@ class PropertyTemplate extends Component {
   }
 }
 
+PropertyTemplate.propTypes = {
+  propertyLabel: PropTypes.string,
+  propertyURI: PropTypes.string,
+  remark: PropTypes.string,
+  mandatory: PropTypes.oneOfType([PropTypes.string, PropTypes.bool]),
```

Comment:
i think right now we have them as strings? are we going to try to change them to booleans in the future?

Reply:
Yes, I thought to maintain compatibility with the JSON Profiles we would validate the prop as a string while allowing the flexibility to typecheck as a boolean in the UI. If the strings `"true"` and `"false"` can recasted as booleans we could set `mandatory` and `repeatable` as just `PropTypes.bool`. Might be a simple one line function!